angular.json

```diff
@@ -9,13 +9,24 @@
       "projectType": "application",
       "architect": {
         "build": {
-          "builder": "@angular-devkit/build-angular:browser",
+          "builder": "@angular-devkit/build-angular:application",
           "options": {
-            "outputPath": "dist/browser",
-            "index": "./site/doc/index.html",
-            "main": "./site/doc/main.ts",
-            "tsConfig": "./site/doc/tsconfig.app.json",
-            "polyfills": "./site/doc/polyfills.ts",
+            "outputPath": "dist",
```

Comment:
With SSR/SSG enabled, the browser and server subfolders are automatically added to the output path.